abtesting/README.md

```diff
@@ -18,7 +18,9 @@ Verify the following two flows:
 Test on device: 
 Run the sample and copy the printed installation auth token from Xcode's console
 into the 'Manage test devices' section in Firebase Console (click into details
-in 'Experiment Overview' when experiment is in Draft status).
+in 'Experiment Overview' when experiment is in Draft status). Verify you can
```

Comment:
How about "Toggle the A/B test state by tapping the Refresh button."

Reply:
Hmmm that's what I was worried about... The button only reads from RemoteConfig and applies changes to the UI, if any. It does not actually switch you out of the A/B test. So repeated presses will log to console
```
Config fetch completed with status: success
Remote Config did not change from: dark
```
for example. How could I make that clearer?

Reply:
I see...  How about something like:

"Tap the Refresh button to update the UI after making changes to the A/B test configuration on the console."